src/Compilers/Core/Portable/Text/TextLine.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
📝 For some reason, when I annotated this file I ended up making the `_text` field nullable. Possibly due to it being a defaultable value type?

Reply:
Good catch. That's the approach I usually take but I forgot it here. 

---
In reply to: [364501192](https://github.com/dotnet/roslyn/pull/40831#discussion_r364501192) [](ancestors = 364501192)